<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>650</height>
     </rect>
    </property>
    <widget class="QWidget" name="startScreen">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/UI/UI/StartScreenBackground.png);</string>
     </property>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>641</width>
        <height>120</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 50px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gameScreen">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/UI/UI/GameScreenBackground.png);</string>
     </property>
     <widget class="QPushButton" name="restartButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="level1Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="level2Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="level3Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="level4Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="howToPlayButton">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>535</y>
        <width>129</width>
        <height>50</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>718</x>
        <y>537</y>
        <width>76</width>
        <height>48</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="statsButton">
      <property name="geometry">
       <rect>
        <x>814</x>
        <y>537</y>
        <width>76</width>
        <height>48</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="hintButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>537</y>
        <width>76</width>
        <height>48</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="popupModal">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>401</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/UI/UI/popupModal.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>339</x>
         <y>23</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="hydrogenButton">
      <property name="geometry">
       <rect>
        <x>718</x>
        <y>135</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="heliumButton">
      <property name="geometry">
       <rect>
        <x>814</x>
        <y>135</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="carbonButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>135</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="nitrogenButton">
      <property name="geometry">
       <rect>
        <x>718</x>
        <y>202</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="oxygenButton">
      <property name="geometry">
       <rect>
        <x>814</x>
        <y>202</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="sodiumButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>202</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="aluminiumButton">
      <property name="geometry">
       <rect>
        <x>718</x>
        <y>269</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="potassiumButton">
      <property name="geometry">
       <rect>
        <x>814</x>
        <y>269</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="calciumButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>269</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ironButton">
      <property name="geometry">
       <rect>
        <x>718</x>
        <y>336</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="nickelButton">
      <property name="geometry">
       <rect>
        <x>814</x>
        <y>336</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="copperButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>336</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="zincButton">
      <property name="geometry">
       <rect>
        <x>718</x>
        <y>403</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="silverButton">
      <property name="geometry">
       <rect>
        <x>814</x>
        <y>403</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="tinButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>403</y>
        <width>76</width>
        <height>47</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hydrogen info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: transparent;
	border-radius: 10px;
}

QPushButton:hover {
    background: rgba(0, 0, 0, 0.3);
}

QPushButton:pressed {
    background: rgba(0, 0, 0, 0.5);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
